package/whiteboard_hypercard/package

```diff
@@ -0,0 +1,30 @@
+# vim: set ft=sh:
+pkgname=whiteboard_hypercard
+pkgdesc="Real-time collaboration, drawing or whiteboarding"
+url=https://github.com/fenollp/reMarkable-tools
+pkgver=0.3.0
+timestamp=2020-09-21T17:59Z
+section=games
+maintainer="Pierre Fenoll <pierrefenoll@gmail.com>"
+license=Apache-2.0
+
+image=rust:v1.1
+source=(https://github.com/fenollp/reMarkable-tools/archive/v0.3.0.zip)
```

Comment:
`whiteboard_hypercard.draft` should be added to the source array, along with a SKIP entry to the `sha256sums`. See https://github.com/toltec-dev/toltec/tree/testing/package/calculator for a reference.

Reply:
Ah yes that makes sense